migrations/20171111135341_v2-to-v3.py

```diff
@@ -0,0 +1,99 @@
+"""
+a caribou migration
+
+name: v2-to-v3
+version: 20171111135341
+"""
+
+def upgrade(connection):
+    # Rename old table
+    sql = """
+        ALTER TABLE trades RENAME TO trades_orig;
+        """
+    connection.execute(sql)
+
+    # Create new table
+    sql = """
+        CREATE TABLE trades (
```

Comment:
I suspect it is a caribou default to add the create table statement here - but isn't this code duplicated to `_DECL_BASE.metadata.create_all(engine)` - which basically does the same thing without specifying the column definition twice?